rre-dataset-generator/src/llm/llm_service.py

```diff
@@ -42,3 +41,49 @@ def generate_queries(self, document: Document, num_queries_generate_per_doc: int
         except JSONDecodeError:
             log.warning("LLM hallucinated and its response: %s", raw)
             return raw
+
+    def generate_scores(self, document: Document, query: str, relevance_scale: str) -> int:
```

Comment:
I would get a type ENUM for the relevance_scale instead of a string parsed and validated directly when we read the config file